library/src/scala/Tuple.scala

```diff
@@ -74,6 +88,16 @@ object Tuple {
     case h *: t => F[h] *: Map[t, F]
   }
 
+  /** Given two tuples, `(A1, ..., An)` and `(B1, ..., Bn)`, returns a tuple
+   *  `((A1, B1), ..., (An, Bn))`. If the two tuples have different sizes,
+   *  the extra elements of the larger tuple will be disregarded.
+   */
```

Comment:
```
  /** Given two tuples, `A1 *: ... *: An * At` and `B1 *: ... *: Bn *: Bt` 
   *  where at least one of `At` or `Bt` is `Unit` or `Tuple`, 
   *  returns the tuple type `(A1, B1) *: ... *: (An, Bn) *: Ct` 
   *  where `Ct` is `Unit` if `At` or `Bt` is `Unit`, otherwise `Ct` is `Tuple`.
   */
```

Reply:
The previous doc assumed the tuples where of fixed size (i.e. had a know tail).